PyAssembler/Assembler.py

```diff
@@ -0,0 +1,421 @@
+#!/usr/bin/env python3.7
+
+import sys
+import struct
+import argparse
+import binascii
+from enum import Enum
+
+
+STR_R_FORMAT = 'r'
+BYTES_W_FORMAT = 'wb'

```

Comment:
These 2 are an example of non-magical constants that can just live in the code without having global constants.

The reason being: `open()`'s _mode_ strings are well known and if someone doesn't remember, they can look it up (and it would be OK).

Note that the constant's name contains the entire string content (STR\_**R**\_FORMAT -> '**r**', **B**YTES\_**W**\_FORMAT -> '**wb**'). This is a sign that something is not quite right.